test/with_dummyserver/test_socketlevel.py

```diff
@@ -70,7 +37,7 @@ def test_hostname_in_first_request_packet(self):
         def socket_handler(listener):
             sock = listener.accept()[0]
 
-            self.buf = sock.recv(65536) # We only accept one packet
+            self.buf = sock.recv(65536)  # We only accept one packet
```

Comment:
@shazow how do you feel about random fixing of pep8/flake8 violations? Do you like two spaces before an in-line comment or not?